### PR TITLE
Create 2019-05-07-coreweekly-week-18-2019.md

### PR DESCRIPTION
**Pull request by ttoine**

### CODE REVIEW

File: news/_posts/core-weekly/2019-05-07-coreweekly-week-18-2019.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+
+If you want to compare, below are the equivalent links for two previous releases:
+
+- [1.7.5.0 milestone](https://github.com/PrestaShop/PrestaShop/milestone/41?closed=1), with 596 items, including [146 issues](https://github.com/PrestaShop/PrestaShop/issues?utf8=%E2%9C%93&q=is%3Aissue+milestone%3A1.7.5.0) and [450 pull requests](https://github.com/PrestaShop/PrestaShop/pulls?q=is%3Apr+milestone%3A1.7.5.0)
+- [1.7.4.0 milestone](https://github.com/PrestaShop/PrestaShop/milestone/34), with 260 pull requests; issues were still managed on the old forge at this time, that explains why there are no figures for issues.
+
+So, now, you know how to check that PrestaShop 1.7.6.0 will be a huge release!
+
+And, that doesn't list the work being done on the module: it is just for the core. You can take some inspiration from the links in the next section of this core weekly to track the work done on the entire project.
```
Comment (matks) [inline news/_posts/core-weekly/2019-05-07-coreweekly-week-18-2019.md]:
```suggestion
And that doesn't include the work carried out for the modules: this is only for the core. You can find some inspiration from the links in the next section of this core weekly to track the work done on the entire project.
```

File: news/_posts/core-weekly/2019-05-07-coreweekly-week-18-2019.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+* [#23](https://github.com/PrestaShop/ps_checkpayment/pull/23): Update Readme & License, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#24](https://github.com/PrestaShop/ps_checkpayment/pull/24): Update Readme & License, by [@eternoendless](https://github.com/eternoendless)
+* [#42](https://github.com/PrestaShop/ps_wirepayment/pull/42): Update Readme & License, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#43](https://github.com/PrestaShop/ps_wirepayment/pull/43): Update Readme & License, by [@eternoendless](https://github.com/eternoendless)
+* [#27](https://github.com/PrestaShop/ps_categorytree/pull/27): Update Readme & License, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#28](https://github.com/PrestaShop/ps_categorytree/pull/28):  Update Readme & License, by [@eternoendless](https://github.com/eternoendless)
+* [#27](https://github.com/PrestaShop/productcomments/pull/27): Update Readme, by [@eternoendless](https://github.com/eternoendless)
+* [#28](https://github.com/PrestaShop/productcomments/pull/28): Add Readme and License, by [@eternoendless](https://github.com/eternoendless)
```
Comment (Quetzacoalt91) [inline news/_posts/core-weekly/2019-05-07-coreweekly-week-18-2019.md]: We deal with different repositories here, what do you think about something like 

```suggestion
* [Product Comments #28](https://github.com/PrestaShop/productcomments/pull/28): Add Readme and License, by [@eternoendless](https://github.com/eternoendless)
```

File: news/_posts/core-weekly/2019-05-07-coreweekly-week-18-2019.md [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+* [#13504](https://github.com/PrestaShop/PrestaShop/pull/13504): Fix backoffice translations for third-party modules, by [@eternoendless](https://github.com/eternoendless)
+* [#13563](https://github.com/PrestaShop/PrestaShop/pull/13563): Do not get data from null currency, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#13576](https://github.com/PrestaShop/PrestaShop/pull/13576): Fix deleting profile . Thank you [@sarjon](https://github.com/sarjon)
+* [#13603](https://github.com/PrestaShop/PrestaShop/pull/13603): Add id for orders view page to ease E2E testing, by [@matks](https://github.com/matks)
+
+
+### Front office
+
+* [#13407](https://github.com/PrestaShop/PrestaShop/pull/13407): Fix labels display computing, by [@jolelievre](https://github.com/jolelievre)
+* [#13440](https://github.com/PrestaShop/PrestaShop/pull/13440): Change checkout behaviour (needs to be validated after meeting), by [@jolelievre](https://github.com/jolelievre)
```
Comment (matks) [inline news/_posts/core-weekly/2019-05-07-coreweekly-week-18-2019.md]:
```suggestion
* [#13440](https://github.com/PrestaShop/PrestaShop/pull/13440): Change checkout behaviour, by [@jolelievre](https://github.com/jolelievre)
```
Maybe we dont need to write this part 😁

File: news/_posts/core-weekly/2019-05-07-coreweekly-week-18-2019.md [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 29th of April to Sunday 5th of May 2019.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+
+## General messages
```
Comment (matks) [inline news/_posts/core-weekly/2019-05-07-coreweekly-week-18-2019.md:17]: Maybe we could explain why there is so much PR that fix tests ? Related to the validation process for 1.7.6 ?

File: news/_posts/core-weekly/2019-05-07-coreweekly-week-18-2019.md (hunk 2 above)
Comment (ttoine) [reply news/_posts/core-weekly/2019-05-07-coreweekly-week-18-2019.md]: good idea. that was to avoir to create a sub section for each module

File: news/_posts/core-weekly/2019-05-07-coreweekly-week-18-2019.md (hunk 4 above)
Comment (ttoine) [reply news/_posts/core-weekly/2019-05-07-coreweekly-week-18-2019.md:17]: That could be a very good news for the next core weekly, or, maybe, a dedicated article.

File: news/_posts/core-weekly/2019-05-07-coreweekly-week-18-2019.md [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+
+
+### OnBoarding
+
+* [#56](https://github.com/PrestaShop/welcome/pull/56): Fix theme & logo, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#57](https://github.com/PrestaShop/welcome/pull/57): Update module version, README and LICENSE, by [@jolelievre](https://github.com/jolelievre)
+* [#58](https://github.com/PrestaShop/welcome/pull/58): Version updated to v6.0.0, by [@jolelievre](https://github.com/jolelievre)
+
+
+### Update of readme.md and license.md
```
Comment (matks) [inline news/_posts/core-weekly/2019-05-07-coreweekly-week-18-2019.md]: Title is a bit strange 🤔 maybe we can explain what this means ?

Comment (ttoine) [reply news/_posts/core-weekly/2019-05-07-coreweekly-week-18-2019.md]: I will improve the title.

Comment (ttoine) [reply news/_posts/core-weekly/2019-05-07-coreweekly-week-18-2019.md]: is it better this way ?

Comment (matks) [reply news/_posts/core-weekly/2019-05-07-coreweekly-week-18-2019.md]: Yes 😄

**Pull request merged**